src/cmd/list.go

```diff
@@ -206,6 +206,7 @@ func getImages() ([]toolboxImage, error) {
 	for _, image := range images {
 		var i toolboxImage
 		var isToolboxImage bool = false
+		var isDuplicatedImage bool = false
```

Comment:
Nitpick: there's no need for the `= false`.  Just this much is enough in Go:
```
var isDuplicatedImage bool
```

I know the (wrong) pattern was already [there](https://github.com/containers/toolbox/pull/469) in the code, so you can use a separate commit to fix it up.